apps/spruce/src/pages/adminSettings/tabs/FeatureFlagsTab/FeatureFlagsTab.tsx

```diff
@@ -0,0 +1,6 @@
+export const FeatureFlagsTab = () => (
+  <div className="flex flex-col gap-4">
+    <h1 className="text-xl font-bold">Feature Flags</h1>
```

Comment:
same comment

also, remove classnames
```suggestion
  <div>
    <h1>Feature Flags</h1>
```